app/models/issue.rb

```diff
@@ -99,8 +99,8 @@ def can_show? cuser
     cuser and !cuser.nil? and ((author == cuser) or cuser.admin?)
   end
 
-  def can_create? cuser, params = {}
-    Verification.contain params, [:title, :category_id, :text]
+  def can_create? cuser
```

Comment:
Unused method argument - `cuser`. If it's necessary, use `_` or `_cuser` as an argument name to indicate that it won't be used. You can also write as `can_create?(*)` if you want the method to accept any arguments but don't care about them.<br>Use def with parentheses when there are parameters.